galileo/Cargo.toml

```diff
@@ -44,6 +44,7 @@ quick_cache = "0.4"
 futures-intrusive = "0.5"
 geojson = { version = "0.24", optional = true }
 raw-window-handle = { version = "0.6", optional = true }
+geozero = { version = "0.12", features = ["with-mvt"] }
```

Comment:
Why do we need this dependency here? And why the version is different from the one in `galileo-mvt`?